client/src/plus/app.tsx

```diff
@@ -48,12 +62,9 @@ export default function App() {
                 technical deep dives written by industry experts and powerful
                 new features to personalize your MDN experience.
               </p>
-
-              {data && data.variant && (
-                <a href="#waitlist" className="button">
-                  Join the waitlist
-                </a>
-              )}
+              <a href="#waitlist" className="button">
```

Comment:
No, the `waitlist` might not load. That's why it *was* dependent on `data && data.variant` before.